DevExpress/DevExtreme
# PivotGrid - ExcelJs Export. Implement KeepColumnWidths property

* Initial realization of keepColumnsWidth for pivotGrid

* Updated tests for a keepColumnWidths option

* Small refactoring

* Fix implementation for datagrid

* Fix test in FF

* Fix terst in FF

* Fix export with server side rendering

* Fix tests for FF

* Tests for field width option

* virtual scrolling support

* Fix rowHeaderLayout for tree value

* Bugfix of undefined value

* Use only static data for tests

* Check multiple grid width

* Fix test for different font rendering engine

* Fix tests for ci environment

* Small fix

* Even more test cases

* Small renaming
